.jshintrc

```diff
@@ -0,0 +1 @@
+{"node": true}
```

Comment:
Can we remove `.jshintrc`? It looks like we are using `eslint` for `grunt lint`.

If we remove, we can remove its entry in `Gruntfile.js`:

    jsonlint: {
        all: ["package.json", ".jshintrc", "src/**/*.json", "tests/**/*.json"]
    }